### PR TITLE
Update install docs for 2.5 release

### PR DESCRIPTION
**Pull request by scollazo**
- Removed apcu_bc install from Xenial install
- Added bionic install doc, based on Xenial one but using php 7.2 and mysql-server-5.7

### CODE REVIEW

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 1]
```diff
@@ -364,56 +288,50 @@ called :file:`/etc/php5/fpm/pool.d/atom.conf`:
...
+   sudo systemctl enable php7.2-fpm
+   sudo systemctl start php7.2-fpm
 
 If the service fails to start, make sure that the configuration file has been
 has been pasted properly. You can also check the syntax by running:
 
 .. code-block:: bash
 
-   sudo php5-fpm --test
+   sudo php-fpm7.2 --test
```
Comment (jraddaoui) [inline admin-manual/installation/linux/ubuntu-bionic.rst:324]: `php7.2-fpm`

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 2]
```diff
@@ -23,39 +23,30 @@ under the `repositories
...
-.. _linux-ubuntu-trusty-dependency-mysql:
+.. _linux-ubuntu-bionic-dependency-mysql:
 
 MySQL
 -----
 
-We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.5 as it's
-much better than the previous major release in terms of speed, scalability and
+We strongly recommend using `MySQL <https://www.mysql.com/>`__  5.7 as it's much
+better than the previous major release in terms of speed, scalability and
```
Comment (jraddaoui) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: They always improve that ;)

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 3]
```diff
@@ -242,64 +217,19 @@ in :file:`/etc/nginx/sites-available/atom`.
...
-      </LocationMatch>
-    </VirtualHost>
-
-You also need to decide if you are going to use php5-fpm or mod_php. We prefer
-the former, in combination with Nginx, but you can combine php5-fpm and Apache
-as long as you install the Apache module mod_fastcgi (the corresponding Ubuntu
-package is called libapache2-mod-fastcgi). We have not tried this ourselves but
-it is definitely `possible <https://wiki.apache.org/httpd/PHP-FPM>`__.
-
-.. _linux-ubuntu-trusty-dependency-php:
```
Comment (jraddaoui) [inline admin-manual/installation/linux/ubuntu-bionic.rst:295]: I think it's okay to remove the Apache section, I'd also remove it from the Xenial docs for consistency.

**Review by jraddaoui — CHANGES_REQUESTED** [review]
Thanks @scollazo, it looks great! Just a couple of comments.

**Review by corinnemrogers — APPROVED** [review]
Looks great! I've suggested a couple of very picky edits - please feel free to accept or disregard them!

File: admin-manual/installation/linux/ubuntu-bionic.rst (hunk 1 above)
Comment (scollazo) [reply admin-manual/installation/linux/ubuntu-bionic.rst:324]: Seems like the command is named `php-fpm7.2 ` :man_shrugging:

File: admin-manual/installation/linux/ubuntu-bionic.rst (hunk 3 above)
Comment (scollazo) [reply admin-manual/installation/linux/ubuntu-bionic.rst:295]: The trusty file was removed, I don't know why git thinks that it was renamed to bionic...

The apache install instructions are not part of the xenial or bionic installs, am I missing something?

File: admin-manual/installation/linux/ubuntu-bionic.rst (hunk 1 above)
Comment (jraddaoui) [reply admin-manual/installation/linux/ubuntu-bionic.rst:324]: :man_facepalming:

File: admin-manual/installation/linux/ubuntu-bionic.rst (hunk 3 above)
Comment (jraddaoui) [reply admin-manual/installation/linux/ubuntu-bionic.rst:295]: I thought it would be also in the Xenial docs, thanks!

**Review by jraddaoui — APPROVED** [review]

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 4]
```diff
@@ -116,7 +105,7 @@ the most popular and we like it, but we've found that
 `Nginx <http://nginx.com/>`__ adapts itself much better to limited resource
 environments while it also scales better and more predictably under high loads.
 You are welcome to try other solutions, but the following documentation will
-focus upon Nginx and Apache, our preferred web server solutions.
+focus merely on Nginx.
```
Comment (corinnemrogers) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: I would remove 'merely': "...focus on Nginx."

File: admin-manual/installation/linux/ubuntu-bionic.rst [hunk 5]
```diff
@@ -242,64 +217,19 @@ in :file:`/etc/nginx/sites-available/atom`.
...
-.. _linux-ubuntu-trusty-dependency-php:
+.. _linux-ubuntu-bionic-dependency-php:
 
 PHP
 ---
 
-You will need to ensure that you have **at least PHP 5.5 or higher installed** 
-- we use PHP 5.5 in development - in addition to a number of PHP extentions,
-described below.
+Ubuntu 18.04 bundles PHP 7.2 which is much faster than older releases.
```
Comment (corinnemrogers) [inline admin-manual/installation/linux/ubuntu-bionic.rst]: add a comma: "... bundles PHP 7.2, which..."

**Review by DavidHume-ArtefactualSystems — COMMENTED** [review]
Realize already merged, which is fine, agree with the changes (and first commit I guess answers my DM question from yesterday! :-) )

Might've asked for this one when we tended to remove previous OS sections per eol, but certainly 'close enough' per

"The standard support period is now nearing its end and Ubuntu 14.04 will transition to
Extended Security Maintenance (ESM) on Thursday, April 25th, 2019."

**Pull request merged**